src/lxc/confile.c

```diff
@@ -4458,6 +4546,42 @@ static int clr_config_cgroup_dir(const char *key, struct lxc_conf *lxc_conf,
 	return 0;
 }
 
+static int clr_config_cgroup_monitor_dir(const char *key,
+					 struct lxc_conf *lxc_conf,
+					 void *data)
+{
+	if (lxc_conf->cgroup_meta.monitor_dir) {
+		free(lxc_conf->cgroup_meta.monitor_dir);
+		lxc_conf->cgroup_meta.monitor_dir = NULL;
+	}
```

Comment:
I think this should just be:
```c
free_disarm(lxc_conf->cgroup_meta.monitor_dir);
```
which does all that. :)

Reply:
Right. I see a cleanup commit for existing cases in the future. ;-)